charts/alertmanager/templates/statefulset.yaml

```diff
@@ -103,6 +103,8 @@ spec:
           resources:
             {{- toYaml .Values.configmapReload.resources | nindent 12 }}
           {{- with .Values.configmapReload.containerPort }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
```

Comment:
Does it make sense to use the same defaults as the main container (runAsNonRoot: true, runAsUser/Group: 65534) or do you think it should get the old default of "not set"?

Reply:
i would check if the old defaults work.
not sure if they use the same user for example.
i guess using it empty would be best. otherwise deployment might break for some users.

Reply:
The old default works fine, and so does the current one in this MR. I set it to empty empty object to be sure its backwards compatible.